src/hooks/use-keydown.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from "react";
```

Comment:
The PR introduces a custom hook, `useKeydown` so that it can be reused for https://github.com/cloudinary-community/photobox/issues/14#issuecomment-2171681975 which will be a separate issue/PR.